packages/web-components/src/tree-view/tree-view.template.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
These can all be removed - the project package.json and the license are managed by the package.